packages/create-jsx-email/starter/templates/starter-template.tsx

```diff
@@ -0,0 +1,101 @@
+import {
```

Comment:
this looks like a copy of the https://github.com/shellscape/jsx-email/blob/main/packages/cli/src/assets/email.mustache file but without mustache. we should probably use a symlink to that file so it's included in our build, and use mustache to populate the values we want to convey to the user. that way we have a single source of truth and only one file to update with changes.

this is also a good case for installing the dependencies for the user, and then using `email create` to create the first template instead of having a file in this package as well.